string operator
string[] objects
geometry_msgs/PoseStamped locationInformation